setup.py

```diff
@@ -117,7 +117,7 @@ def build_extension(self, ext):
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6'
+        'Programming Language :: Python :: 3.7'
```

Comment:
feels like this list should reflect the Python versions we're actively testing against.

also, should we keep both 3.6 and 3.7?

Reply:
updated supported languages and added back py36 test